src/main/java/com/erstudio/ERSUI.java

```diff
@@ -0,0 +1,245 @@
+package main.java.com.erstudio;
+
+
+/**
+ * @author akshit.arora
+ * for the UI Skeleton
+ *
+ */
+
+import javax.swing.*;
+import javax.swing.filechooser.FileSystemView;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class ERSUI {
+
+    JLabel labelTSString, labelVersionString, labelNewVersionString, labelGUID;
+    JTextField textFieldTs, textVersionString, textNewVersionString, textGUID;
+    JButton buttonTS, buttonGenerate, jButton, jButton1;
+    ButtonGroup buttonGroup;
+    JRadioButton jRadioButton, jRadioButton1;
+    JTextArea textArea;
+    JFrame frame;
+    JTabbedPane tp;
+    JPanel panel;
+
+    public JPanel initTeamServerUI() {
+        panel = new JPanel();
+        panel.setLayout(new GridBagLayout());
+
+        labelTSString = new JLabel("Please choose TS main folder");
```

Comment:
All the string constants needs to be in separate file with final static.

Reply:
done